tensorflow/core/framework/dataset.h

```diff
@@ -162,7 +162,7 @@ class GraphDefBuilderWrapper {
                     const std::vector<std::pair<StringPiece, AttrValue>>& attrs,
                     Node** output) {
     std::vector<std::pair<size_t, Node*>> enumerated_inputs(inputs.size());
-    for (int i = 0; i < inputs.size(); i++) {
+    for (int i = 0; static_cast<size_t>(i) < inputs.size(); i++) {
```

Comment:
Let's use the following simpler code here:

```suggestion
    for (size_t i = 0; i < inputs.size(); i++) {
```

Reply:
Sure, that makes more sense :smile: